cltk/text_reuse/automata.py

```diff
@@ -118,13 +122,13 @@ def __init__(self, Q, S, s, F, delta = False):
         self.S = S
 
         if s not in self.Q:
-            print("The specified value is invalid, s must be a member of Q")
+            LOG.info("The specified value is invalid, s must be a member of Q")
```

Comment:
Yes, your intuition is correct here, please use LOG.error(...)

Sorry for my miscommunication; same for the other instances below where a "raise" statement follows.